web-app/CKS.Web.Test/IntegrationTests/404Tests.cs

```diff
@@ -0,0 +1,28 @@
+using CKS.Web.Test.IntegrationTests.Infrastructure;
+using Shouldly;
+using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace CKS.Web.Test.IntegrationTests
+{
+	public class NotFoundTests : TestBase
+	{
+		public NotFoundTests(ITestOutputHelper output)
+			: base(output)
+		{
+		}
+
+		[Fact]
+		public async void Returns404Page()
+		{
+			// Arrange
+			var client = _factory.CreateClient();
+
+			// Act
+			var response = await client.GetAsync("/404");
```

Comment:
Can this test be hitting a random not existent route like /xyz and then assert that the status code is not found and assert something about the page content - ie to check that our 404.html page is being served, and not the default .net one. We might need to put a 404.html in the wwwroot folder to be able to assert against this 'fake content', assuming this file will be overwritten with the real 404.html, generated from the gatsby site.